client/stylesheets/embed.css

```diff
@@ -0,0 +1,105 @@
+.embedContainer {
+	border: 1px solid;
+    border-color: #e5e6e9 #dfe0e4 #d0d1d5;
+    border-radius: 3px;
+    display: flex;
+  	flex-direction: column;
+  	font-size: 16px;
+}
+
+.embedAce {
+	flex: 1;
+	display: flex;
+}
+
+#embedEditor {
+	flex: 1;
+	border-radius: 3px 3px 0 0;
+}
+
+.footerContainer {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	background-color: #EEE;
+	border-radius: 0 0 3px 3px;
+}
+
+.footerContainer > div {
+	margin: 16px;
+}
+
+.embedLang {
+	height: 16px;
+}
+
+.embedDialogContainer {
+	z-index: 10;
+	position: fixed;
+	display: none;
+	opacity: 0;
+	justify-content: center;
+	align-items: center;
+	pointer-events: none;
+	width: 100%;
+	height: 100%;
+	top: 0;
+	left: 0;
+	animation-duration: 0.3s;
+	animation-fill-mode: forwards;
+}
+
+.embedDialogOverlay {
+	pointer-events: auto;
+	opacity: 0;
+	width: 100%;
+	height: 100%;
+	top: 0;
+	left: 0;
+	z-index: 10;
+	position: absolute;
+}
+
+.embedDialog {
+	pointer-events: auto;
+	width: 70%;
+	max-width: 560px;
+	min-width: 290px;
+	background: #fff;
+	padding: 4em;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using width with padding can sometimes make elements larger than you expect.](https://www.codacy.com/app/igougi-ui/Mrb-in/file/2618786806/issues/source?bid=2882324&fileBranchId=3313627#l70)